### ISSUE TITLE
Great work , 

### ISSUE THREAD

**Issue by saeed7m** [UniSharp/laravel-filemanager#225]
Great work , 
what happen when i upload some image with my FTP , the thumb not shown , icon instead , i changed 
$thumb_path = $this->getThumbPath($file_name);
```php
if (File::exists($thumb_path)) {
    $thumb_url = $this->getThumbUrl($file_name) . '?timestamp=' . filemtime($thumb_path);
} else {
    $thumb_url = null;
}
```

to 
```php
$thumb_path = $this->getThumbPath($file_name);
if (File::exists($thumb_path)) {
    $thumb_url = $this->getThumbUrl($file_name) . '?timestamp=' . filemtime($thumb_path);
} else {
    $thumb_url = $this->getFileUrl($file_name)  ;
}
```

now ,when thumb not available ,image show as thumbnail, 
do you have a better solution

**Comment by g0110280** [comment]
Image icons will be shown when thumbnails are not available. Maybe a new config can be added in future versions.

**Comment by g0110280** [comment]
Implemented in `v1.7.6`.
